### PR TITLE
2775 fix typo

### PR DESCRIPTION
**Pull request by droffilc1**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/semantic-html/terms/ruby/ruby.md
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/ruby/ruby.md:1]: Remove this file from this PR.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@droffilc1 only one comment. Remove the extra file from the prior PR.

File: content/html/concepts/semantic-html/terms/ruby/ruby.md
Comment (droffilc1) [reply content/html/concepts/semantic-html/terms/ruby/ruby.md:1]: Okay.

**Review by SSwiniarski — APPROVED** [review]
@droffilc1 Looks good. Sending it for review two.

**Review by caupolicandiaz — APPROVED** [review]
@droffilc1 @SSwiniarski LGTM

**Pull request merged**